src/kvirc/ui/KviWindow.h

```diff
@@ -148,13 +148,12 @@ class KVIRC_API KviWindow : public QWidget
 		UserWindow = 16,
 		Tool = 17,
 		IOGraph = 18,
-		DirBrowser = 19,
-		ScriptEditor = 20,
-		ScriptObject = 21,
-		LogView = 22,
-		Offer = 23,
-		Debug = 24,
-		Unknown = 25,
+		ScriptEditor = 19,
+		ScriptObject = 20,
+		LogView = 21,
+		Offer = 22,
+		Debug = 23,
+		Unknown = 24,
 		TypeCount = 26
```

Comment:
My local has count 25, I guess that would be right then. but since we have to comment these out Ill revert and leave the trash behind